schemas/com.snowplowanalytics.monitor/status_report/jsonschema/1-0-0

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a status report event emitted from an application",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitor",
+    "name": "status_report",
```

Comment:
@dilyand you prefer the name `status_report` to `report_my_status`?

I'm not tied to `report_my_status` particularly but keen to name it something that makes it clear this is the event - `report_status` sounds like a noun rather than a verb...

Reply:
No strong feelings, but I followed the Tracking Conventions drafted by Implementation Engineering:

> Custom event names to read `(object_)action` (with the exception of prefixes in next section), for example:
>- `button_click`
>- `nativead_impression`
>- `view`
>- Note: NOT `view_event`

So: `status_report`.

Reply:
👍